tests/test_config.py

```diff
@@ -197,6 +198,7 @@ def test_public_attrs(self):
         """Assert that public attributes have correct values."""
         attrs = config._public_attrs(self.cfg)  # pylint:disable=W0212
         attrs['pulp_version'] = type('')(attrs['pulp_version'])
+        attrs['selinux_enabled'] = type(True)(attrs['selinux_enabled'])
```

Comment:
`type(True)` can be replaced with `bool`. The `type('')` on the line above is a hold-over from when both Python 2 and Python 3 were supported, and `''` might be either a byte or a string.

Reply:
Ack. I wasn't sure, I thought the construct was odd.